react-loops.js

```diff
@@ -79,19 +89,28 @@ function For(props) {
       "<For> expects either an Iterable `of` or Object `in` prop."
     );
   }
-
+  var hasFallback = props.hasOwnProperty('fallback');
+  if(hasFallback) {
+    var fallback = props.fallback;
+  }
   if (hasIn) {
     // Get the object
     var obj = props.in;
 
     // Accept null / falsey as nothing to loop.
     if (!obj) {
+      console.warn(
```

Comment:
In general I like to avoid direct use of `console.warn` in library code. It's easy to miss and when you are looking at your console it could lead to a lot of spew.

Also, the warning pattern is best saved for communicating deprecated usage patterns, and I didn't understand this PR's proposal as deprecating `null` behaving the same as a empty collection.

Reply:
I can't agree with your point of view anymore. I will fix this

Reply:
In fact, I am a newbie and the first time I submitted the code. Thank you for your code review, I will modify it according to the suggestions.